packages/presentation/lib/base/bloc.dart

```diff
@@ -38,7 +38,5 @@ abstract class BlocImpl<T extends BaseArguments, D> implements Bloc<T, D> {
   void initArgs(T arguments) {}
 
   @override
-  void dispose() {
-    _data.close();
-  }
+  void dispose() {}
```

Comment:
why did u remove closing the stream?

Reply:
it conflicted with `_data.add(_blocData.copy<D>());`

Reply:
[ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: Bad state: Cannot add event after closing

Reply:
need to resolve the error. seems that u try to push event after dispose